rasterio/merge.py

```diff
@@ -170,6 +168,7 @@ def merge(sources, bounds=None, res=None, nodata=None, precision=7):
 
         np.copyto(
             region, temp,
-            where=np.logical_and(region == nodataval, temp.mask == False))
+            where=np.logical_and(region == nodataval,
+                                 np.logical_not(temp.mask)))
```

Comment:
@perrygeo use of `np.logical_not` eliminates a linter warning that always bugged me.